openssl-sys/src/ssl.rs

```diff
@@ -406,6 +408,10 @@ cfg_if! {
         }
     }
 }
+#[cfg(ossl102)]
+pub unsafe fn SSL_add_chain_certificate_pem(ssl: *mut ::SSL, ptr: *mut c_void) -> c_long {
```

Comment:
Does this exist as a macro in OpenSSL? Google does not return any results for `SSL_add_chain_certificate_pem`.

Reply:
It looks like it's called `SSL_add1_chain_cert` instead: https://github.com/openssl/openssl/blob/269ad8d571e68513175fdc66227943786353cfa8/include/openssl/ssl.h.in#L1410

Reply:
yes, it's a macro. I have rename it to reflect its macro name. but retain pem suffix